pilot/pkg/proxy/envoy/v2/gen2.go

```diff
@@ -116,6 +123,7 @@ func (s *DiscoveryServer) handleCustomGenerator(con *XdsConnection, req *xdsapi.
 	w.LastSent = time.Now()
 	w.LastSize = sz // just resource size - doesn't include header and types
 	w.NonceSent = resp.Nonce
+	adsLog.Infoa("Pushed ", w.TypeUrl, " count=", len(cl), " sz=", sz)
```

Comment:
Can you follow the same logging format as others? like `adsLog.Infof("CDS: PUSH for node:%s clusters:%d services:%d version:%s",`. makes it easier to parse

Reply:
Sure.